conf/defaults.ini

```diff
@@ -1013,8 +1013,8 @@ default_timezone = browser
 enabled = true
 
 [geomap]
-# Set the default base layer in GeoMaps plugin
-default_baselayer =
+# Set the JSON configuration for the default basemap
+default_baselayer_config =
```

Comment:
looks a bit odd with nothing on the right side, should we put empty string?

Reply:
in a `.ini` file this is how you specify an empty string